### DISCUSSION TITLE
Does Ray support nuxt 2 ?

### DISCUSSION THREAD

**Discussion by lrandom** [permafrost-dev/vue-ray#37] [Q&A] [answered]
Hi 
I have a question 
Does Ray support nuxt 2 ?
Best Regard
Luan

**Comment by patinthehat** [comment, marked as answer]
@lrandom I haven't tried it with Nuxt specifically, but it should work just fine.

**Reply by lrandom** [reply]
Thanks, It working great on Nuxt but can you pls add a button to clear log on Ray App. I already purchased your product for 2 year, it perfect app. 
Best Regards.
Luân
